### PR TITLE
docs(core): add autodocs for keys and key-sets

### PR DESCRIPTION
**Pull request by jschmid1**
### Summary

* Add autodocs for keys and key-sets 

Related docs PRs

* https://github.com/Kong/docs.konghq.com/pull/4804
* https://github.com/Kong/docs.konghq.com/pull/4802

Signed-off-by: Joshua Schmid <jaiks@posteo.de>

### CODE REVIEW

File: scripts/autodoc [hunk 1]
```diff
@@ -176,7 +176,7 @@ done
 
 insert_yaml_file_into_nav_file ./autodoc/output/nav/docs_nav.yml.admin-api.in \
                                "$DOCS_NAV" \
-                               "/title: Admin API/="
+                               "/text: Admin API/="
```
Comment (jschmid1) [inline scripts/autodoc:179]: There seems to be a change in structure which isn't consistent across different docs version. Can someone from the @Kong/docs-maintainers give some context heer?

File: autodoc/admin-api/data/admin-api.lua [hunk 2]
```diff
@@ -2102,6 +2104,151 @@ return {
...
+      entity_title = "Key Set",
+      entity_title_plural = "Key Sets",
+      description = [[
+        An Key Set object holds a collection of asymmetric key objects.
+        This entity allows to logically group keys by their purpose.
+      ]],
+      fields = {
+        id = { skip = true },
+        created_at = { skip = true },
+        updated_at = { skip = true},
```
Comment (chronolaw) [inline autodoc/admin-api/data/admin-api.lua:2118]: style: `{ skip = true }`

File: autodoc/admin-api/data/admin-api.lua [hunk 3]
```diff
@@ -2102,6 +2104,151 @@ return {
...
+            An optional set of strings associated with the Key for grouping and filtering.
+          ]],
+          examples = {
+            { "google-keys", "mozilla-keys" },
+            { "production", "staging", "development" }
+          },
+        },
+      },
+      ["/key-sets/:key_sets"] = {
+        -- needs this to be present because there is a nested enpoint like `key-sets/:id/keys`
```
Comment (chronolaw) [inline autodoc/admin-api/data/admin-api.lua:2131]: typo: `endpoint`

File: autodoc/admin-api/data/admin-api.lua [hunk 4]
```diff
@@ -2102,6 +2104,151 @@ return {
...
+      entity_title_plural = "Keys",
+      description = [[
+        A Key object holds a representation of asymmetric keys in various formats.
+        When Kong or a Kong plugin requires a specific public or private key to perform
+        certain operations, it can use this entity.
+      ]],
+      fields = {
+        id = { skip = true },
+        created_at = { skip = true },
+        updated_at = { skip = true},
```
Comment (chronolaw) [inline autodoc/admin-api/data/admin-api.lua:2205]: style: `{ skip = true }`

File: autodoc/admin-api/data/admin-api.lua [hunk 5]
```diff
@@ -2102,6 +2104,151 @@ return {
...
+        name = { description = [[The name to associate with the given keys.]] },
+        set = {
+          description = [[
+            The id (a UUID) of the key-set with which to associate the key.
```
Comment (chronolaw) [inline autodoc/admin-api/data/admin-api.lua:2209]: typo: `an UUID` ?

File: autodoc/admin-api/data/admin-api.lua [hunk 6]
```diff
@@ -2102,6 +2104,151 @@ return {
...
+          ]]
+        },
+        kid = {
+          description = [[
+            A unique identifier for a key.
+          ]],
+          example = "42"
```
Comment (chronolaw) [inline autodoc/admin-api/data/admin-api.lua:2216]: add a `,` at the end of line?

File: CHANGELOG.md [hunk 7]
```diff
@@ -91,6 +91,8 @@
   [#9611](https://github.com/Kong/kong/pull/9611)
 - Add support for dynamically changing the log level
   [#9744](https://github.com/Kong/kong/pull/9744)
+- Add `keys` entity to store and manage asymmetric keys.
+- Add `key-sets` entity to group and manage `keys`
```
Comment (chronolaw) [inline CHANGELOG.md:95]: Should add PR links for them?

**Pull request merged**